scripts/bin/base_nas_fanout_init.py

```diff
@@ -18,13 +18,11 @@
 import event_log as ev
```

Comment:
Revert the changes in this file. This would break https://github.com/open-switch/opx-nas-interface/commit/78168e4e69fc99ca635dea9962d453a886d1e94c

Reply:
Please address this.